scripts/fast-reboot

```diff
@@ -247,7 +247,17 @@ function backup_database()
     # Delete keys in stateDB except FDB_TABLE|*, MIRROR_SESSION_TABLE|*, WARM_RESTART_ENABLE_TABLE|*, FG_ROUTE_TABLE|*
     sonic-db-cli STATE_DB eval "
         for _, k in ipairs(redis.call('keys', '*')) do
-            if not string.match(k, 'FDB_TABLE|') and not string.match(k, 'WARM_RESTART_TABLE|') \
+            if string.match(k, 'PORT_TABLE|Ethernet') then
```

Comment:
@zjswhhh do you think keeping port_table intact in state-db will cause any change in mux init logic for dualtor warmboot?